### PR TITLE
WV-2193: Use monospace font everywhere we show a date

### PR DESCRIPTION
**Pull request by jasontk19**
## Description

* Update fonts everywhere a date is shown to be monospace

## How To Test

Check the following places for proper font use and any possible style disturbances from these change.  Only the places that were affected by these changes are listed as some other parts of the app were already using monospace fonts.

* Layer info in sidebar
  * Check layers with no dates, only start date, and both start/end dates
* Layer info in product picker (search & measurement browse layer descriptions)
* Event list (individual events and each date within multi-point event)
* Layer coverage panel
* Compare mode:
  * A/B tabs in sidebar
  * Dates on either side of swipe line
  * Dates on either side of opacity slider
  * Date on spy circle
* GIF capture dialog
* GIF results dialog

@nasa-gibs/worldview

### CODE REVIEW

**Review by minniewong — APPROVED** [review]
I think it looks pretty nice in most instances! They do look a little like buttons in the Add Layers view with the dark grey dates over the light grey background, but I'm not sure there's a better alternative.

**Comment by minniewong** [comment]
Did you want to do this date too? 
Screen Shot 2022-02-23 at 4 18 45 PM

**Comment by jasontk19** [comment]
> Did you want to do this date too? Screen Shot 2022-02-23 at 4 18 45 PM

This one gets tricky because of how we use a 3rd party plugin for the facets that takes a string and not a DOM element (which I could give a class or style).  I could style the date in the "chip" above pretty easily but then they would be inconsistent which might look _more_ weird?

**Comment by minniewong** [comment]
> > Did you want to do this date too? Screen Shot 2022-02-23 at 4 18 45 PM
> 
> This one gets tricky because of how we use a 3rd party plugin for the facets that takes a string and not a DOM element (which I could give a class or style). I could style the date in the "chip" above pretty easily but then they would be inconsistent which might look _more_ weird?

I was not actually bothered by it, I just noticed it. I think it looks fine as is :)

**Review by Benjaki2 — APPROVED** [review]
👍

File: web/js/containers/sidebar/compare.js [hunk 1]
```diff
@@ -21,6 +21,8 @@ const CompareCase = (props) => {
 
   const outerClass = 'layer-container sidebar-panel';
   const tabClasses = 'ab-tab';
+  const FormattedDate = ({ date }) => (<span className="monospace">{date}</span>);
```
[the remark this answers is not in the document]
Comment (jasontk19) [reply web/js/containers/sidebar/compare.js:24]: Yeah I like that, meant to go back and do this then forgot.  Will do it now!

**Pull request merged**